source/frontend/src/mocks/fakeISSResult.ts

```diff
@@ -0,0 +1,193 @@
+import { FeatureCollection, Geometry } from 'geojson';
+
+import { ISS_ProvincialPublicHighway } from '../models/layers/pimsHighwayLayer';
+
+const getFakeISSResult: () => FeatureCollection<Geometry, ISS_ProvincialPublicHighway> = () => ({
```

Comment:
nit: we have been calling these functions "getMock..." instead of "getFake..." - would appreciate keeping it consistent